unicodetools/data/emoji/13.1/emoji-zwj-sequences.txt

```diff
@@ -1169,15 +1371,19 @@
 
 # RGI_Emoji_ZWJ_Sequence: Other
 
+2764 FE0F 200D 1F525                        ; RGI_Emoji_ZWJ_Sequence  ; heart on fire                                                  # E0.0   [1] (❤️‍🔥)
+2764 FE0F 200D 1FA79                        ; RGI_Emoji_ZWJ_Sequence  ; mending heart                                                  # E0.0   [1] (❤️‍🩹)
```

Comment:
From Jennifer's comment on the thread "Draft v13.1 charts and data" it sounds like this is supposed to be removed.